coops-ui/tests/unit/StandaloneDirectorsFiling.spec.ts

```diff
@@ -440,7 +445,7 @@ describe('Standalone Directors Filing - Part 3 - Submitting', () => {
       // TODO: verify that new filing was created
 
       const button = wrapper.find('#cod-file-pay-btn')
-      expect(button.attributes('disabled')).toBeUndefined()
+      expect(wrapper.find('#cod-file-pay-btn').attributes('disabled')).not.toBe('true')
```

Comment:
can use the constant "button" from previous line rather than wrapper.find() again.

Reply:
Same on line 512.

Reply:
Good catch, thank you!